types/react-native-material-ui/index.d.ts

```diff
@@ -0,0 +1,657 @@
+    searchable?: Searchable;
+    onPress?(): void;
+    onLeftElementPress?(): void;
+    onRightElementPress?(): void;
+}
+
+/**
+ * Toolbar
+ *
+ * @export
+ * @class Toolbar
+ * @extends {Component<ToolbarProps, any>}
+ * @see https://github.com/xotahal/react-native-material-ui/blob/master/src/Toolbar/Toolbar.react.js
+ */
+export class Toolbar extends Component<ToolbarProps, any> {}
+
+export interface SnackbarProps {
+    message: string;
+    visible: boolean;
+    timeout: number;
+    bottomNavigation: boolean;
+    actionText?: string;
+    button?: ButtonProps;
+    style?: {
+        container?: ViewStyle
+        message?: ViewStyle
+    };
+    onRequestClose(): void;
+    onActionPress?(): void;
+}
+
+/**
+ * Snackbar
+ *
+ * @export
+ * @class Snackbar
+ * @extends {Component<SnackbarProps, any>}
+ * @see https://github.com/xotahal/react-native-material-ui/blob/master/src/Snackbar/Snackbar.react.js
+ */
+export class Snackbar extends Component<SnackbarProps, any> {}
+
+export interface RippleFeedbackProps {
+    color?: string;
+    borderless?: boolean;
+    children: JSX.Element;
+}
+
+/**
+ * Ripple Feedback
+ *
+ * @export
+ * @class RippleFeedback
+ * @extends {Component<IRippleFeedback, any>}
+ * @see https://github.com/xotahal/react-native-material-ui/blob/master/src/RippleFeedback/RippleFeedback.react.js
+ */
+export class RippleFeedback extends Component<RippleFeedbackProps, any> {}
+
+export interface ThemeProviderProps {
+    uiTheme: {};
+    children: JSX.Element;
+}
+
+/**
+ * ThemeProvider
+ *
+ * @export
+ * @class ThemeProvider
+ * @extends {Component<ThemeProviderProps, any>}
+ * @see https://github.com/xotahal/react-native-material-ui/blob/master/src/styles/ThemeProvider.react.js
+ */
+export class ThemeProvider extends Component<ThemeProviderProps, any> {}
+
+export interface MKColorStatic {
+    Amber: string;
+    Blue: string;
+    BlueGrey: string;
+    Brown: string;
+    Cyan: string;
+    DeepOrange: string;
+    DeepPurple: string;
+    Green: string;
+    Grey: string;
+    Indigo: string;
+    LightBlue: string;
+    LightGreen: string;
+    Lime: string;
+    Orange: string;
+    Pink: string;
+    Purple: string;
+    RGBIndigo: string;
+    RGBPink: string;
+    RGBPurple: string;
+    RGBTeal: string;
+    Red: string;
+    Silver: string;
+    Teal: string;
+    Transparent: string;
+    Yellow: string;
+    default: {
+      Amber: string;
+      Blue: string;
+      BlueGrey: string;
+      Brown: string;
+      Cyan: string;
+      DeepOrange: string;
+      DeepPurple: string;
+      Green: string;
+      Grey: string;
+      Indigo: string;
+      LightBlue: string;
+      LightGreen: string;
+      Lime: string;
+      Orange: string;
+      Pink: string;
+      Purple: string;
+      RGBIndigo: string;
+      RGBPink: string;
+      RGBPurple: string;
+      RGBTeal: string;
+      Red: string;
+      Silver: string;
+      Teal: string;
+      Transparent: string;
+      Yellow: string;
```

Comment:
Good question, I have no clue as I have no insight in the bot. You should ping one of the MS people.